apps/nextjs-website/src/components/atoms/ProductHeader/ProductHeader.tsx

```diff
@@ -1,52 +1,29 @@
-import { Path } from '@/lib/types/path';
+import { productToMenuItems } from '@/helpers/productHeader.helper';
 import { Product } from '@/lib/types/product';
 import { Divider, useTheme } from '@mui/material';
 import { Header } from '@pagopa/pagopa-editorial-components/dist/components/Header';
-import { Theme } from '@pagopa/pagopa-editorial-components/dist/types/components';
-import React, { Fragment, FC } from 'react';
+import React, { FC } from 'react';
 
 type ProductHeaderProps = {
   product: Product;
-  currentSlug: string;
+  path: string;
 };
 
-function productToMenuItems(
-  product: Product,
-  currentSlug: string,
-  theme: Theme
-): {
-  href: string;
-  label: string;
-  active: boolean;
-  theme: Theme;
-}[] {
-  return Object.entries(product.subpaths)
-    .filter(([name, subpath]: [string, Path]) => !!name && !!subpath)
-    .map(([name, subpath]: [string, Path]) => {
-      return {
-        label: subpath.name || name,
-        href: subpath.path,
-        active: currentSlug === subpath.path,
-        theme,
-      };
-    });
-}
-
-const ProductHeader: FC<ProductHeaderProps> = ({ product, currentSlug }) => {
+const ProductHeader: FC<ProductHeaderProps> = ({ product, path }) => {
   const { palette } = useTheme();
   const themeVariant = palette.mode;
   return (
-    <Fragment>
+    <>
       <Header
-        menu={productToMenuItems(product, currentSlug, themeVariant)}
+        menu={productToMenuItems(product, path, themeVariant).concat()}
```

Comment:
```suggestion
        menu={[...productToMenuItems(product, path, themeVariant)]}
```
same as comment of another PR